modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -291,6 +291,11 @@ class Editor implements EditorObservable {
       DOMUtils.DOM.styleSheetLoader._setReferrerPolicy(referrerPolicy);
     }
 
+    const contentCssCors = Options.hasContentCssCors(self);
+    if (contentCssCors !== undefined) {
```

Comment:
Should this condition also check that the mode is inline? Or would this be fine to include in classic mode as well?

Reply:
I think it is fine to not check the editor mode as we don't have any specific checks for `referrerPolicy` above either. Could we maybe do `Type.isNonNullable` instead?

Reply:
Good idea 👍